Core/src/main/java/preferences/Get.java

```diff
@@ -45,6 +47,7 @@ public JobHistory jobHistory() {
                 jobHistory.addJob(job, true);
             }
         } catch (SQLException e) {
+             log.error("Error fetching completed jobs", e);
```

Comment:
Fix the indentation to solve linter warnings
 
```suggestion
            log.error("Error fetching completed jobs", e);
```